nexus-sled-agent-shared/src/inventory.rs

```diff
@@ -124,6 +127,26 @@ pub enum SledRole {
     Scrimlet,
 }
 
+/// Describes the set of Reconfigurator-managed configuration elements of a sled
+// TODO this struct should have a generation number; at the moment, each of
```

Comment:
I know you're probably working through this right now, but can we file an issue just in case this gets deferred?

Reply:
Sure thing: #7774